packages/frontend-shared/src/gql-components/modals/CreateCloudOrgModal.vue

```diff
@@ -122,7 +123,7 @@ onBeforeUnmount(() => {
   window.clearTimeout(timer)
 })
 
-const createOrgUrl = computed(() => props.gql.createCloudOrganizationUrl || '#')
```

Comment:
We never want a dead link, better to hide the link if it is somehow not provided in gql.